INFO     core.jenkins_utils:jenkins_utils.py:197 running cleanup
DEBUG    urllib3.connectionpool:connectionpool.py:241 Starting new HTTP connection (1): localhost:8080
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET / HTTP/1.1" 403 None
DEBUG    core.jenkins_utils:jenkins_utils.py:86 failed to get page on the first try, retrying after login
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET /login?from=%2F HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "POST /j_spring_security_check HTTP/1.1" 302 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET / HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET / HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET /me/my-views/view/all/ HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "POST /job/Test_Folder/doDelete HTTP/1.1" 302 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET / HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET /manage/securityRealm/ HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET /computer/ HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET /manage/credentials/store/system/ HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET / HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "POST /submitDescription HTTP/1.1" 302 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET / HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "POST /me/my-views/view/all/submitDescription HTTP/1.1" 302 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET /me/my-views/view/all/ HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "POST /user/admin/appearance/configSubmit HTTP/1.1" 200 133
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET /user/admin/security/ HTTP/1.1" 200 None
DEBUG    conftest:conftest.py:57 Argument --remote-allow-origins=* added to the chrome
DEBUG    conftest:conftest.py:57 Argument --disable-gpu added to the chrome
DEBUG    conftest:conftest.py:57 Argument --no-sandbox added to the chrome
DEBUG    conftest:conftest.py:57 Argument --disable-dev-shm-usage added to the chrome
DEBUG    conftest:conftest.py:57 Argument --headless=new added to the chrome
DEBUG    conftest:conftest.py:57 Argument --window-size=1920,1080 added to the chrome
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:101 Selenium Manager binary found at: /opt/hostedtoolcache/Python/3.12.10/x64/lib/python3.12/site-packages/selenium/webdriver/common/linux/selenium-manager
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:114 Executing process: /opt/hostedtoolcache/Python/3.12.10/x64/lib/python3.12/site-packages/selenium/webdriver/common/linux/selenium-manager --browser chrome --debug --language-binding python --output json
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 Found chromedriver 136.0.7103.92 in PATH: /usr/bin/chromedriver
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 chrome detected at /usr/bin/google-chrome
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 Running command: /usr/bin/google-chrome --version
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 Output: "Google Chrome 136.0.7103.92 "
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 Detected browser: chrome 136.0.7103.92
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 Required driver: chromedriver 136.0.7103.113
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 Driver path: /usr/bin/chromedriver
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 Browser path: /usr/bin/google-chrome
DEBUG    selenium.webdriver.common.service:service.py:231 Started executable: `/usr/bin/chromedriver` in a child process with pid: 9866 using 0 to output -3
DEBUG    urllib3.connectionpool:connectionpool.py:241 Starting new HTTP connection (1): localhost:60127
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:60127 "POST /session HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:60127 "POST /session/25f68880dc87aabc585fa24771588f40/url HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:60127 "GET /session/25f68880dc87aabc585fa24771588f40/url HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:60127 "POST /session/25f68880dc87aabc585fa24771588f40/element HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:60127 "POST /session/25f68880dc87aabc585fa24771588f40/element/f.A5818C05F378E5E09474B99C208F9AA5.d.0670AE27E8E3B35D46981B4A696820E0.e.2/value HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:60127 "POST /session/25f68880dc87aabc585fa24771588f40/element HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:60127 "POST /session/25f68880dc87aabc585fa24771588f40/element/f.A5818C05F378E5E09474B99C208F9AA5.d.0670AE27E8E3B35D46981B4A696820E0.e.3/value HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:60127 "POST /session/25f68880dc87aabc585fa24771588f40/element HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:60127 "POST /session/25f68880dc87aabc585fa24771588f40/element/f.A5818C05F378E5E09474B99C208F9AA5.d.0670AE27E8E3B35D46981B4A696820E0.e.6/click HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:60127 "GET /session/25f68880dc87aabc585fa24771588f40/url HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:60127 "GET /session/25f68880dc87aabc585fa24771588f40/url HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:60127 "GET /session/25f68880dc87aabc585fa24771588f40/cookie HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:241 Starting new HTTP connection (1): localhost:8080
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET /crumbIssuer/api/json HTTP/1.1" 200 155
DEBUG    core.jenkins_utils:jenkins_utils.py:26 update_crumb: 801ca7370408a0eeddf70b6e41592d987e4890afaab724b5310a9bad4dfb6b33
DEBUG    LoginPage:login_page.py:32 login crumb: 801ca7370408a0eeddf70b6e41592d987e4890afaab724b5310a9bad4dfb6b33
DEBUG    MainPage:ui_element.py:53 Click on locator ('link text', 'Manage Jenkins')
DEBUG    MainPage:ui_element.py:28 waiting for element_to_be_clickable(('link text', 'Manage Jenkins'),)
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:60127 "POST /session/25f68880dc87aabc585fa24771588f40/element HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:60127 "POST /session/25f68880dc87aabc585fa24771588f40/execute/sync HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:60127 "GET /session/25f68880dc87aabc585fa24771588f40/element/f.A5818C05F378E5E09474B99C208F9AA5.d.2D0AB59EC4210DD403E02B4E7A7B5669.e.46/enabled HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:60127 "POST /session/25f68880dc87aabc585fa24771588f40/element/f.A5818C05F378E5E09474B99C208F9AA5.d.2D0AB59EC4210DD403E02B4E7A7B5669.e.46/click HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:60127 "GET /session/25f68880dc87aabc585fa24771588f40/url HTTP/1.1" 200 0
DEBUG    ManageJenkinsPage:ui_element.py:53 Click on locator ('xpath', "//a[@href='load-statistics']/..")
DEBUG    ManageJenkinsPage:ui_element.py:28 waiting for element_to_be_clickable(('xpath', "//a[@href='load-statistics']/.."),)
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:60127 "POST /session/25f68880dc87aabc585fa24771588f40/element HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:60127 "POST /session/25f68880dc87aabc585fa24771588f40/execute/sync HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:60127 "GET /session/25f68880dc87aabc585fa24771588f40/element/f.A5818C05F378E5E09474B99C208F9AA5.d.B78F4AFEB072FF7A7008872CAE8C1035.e.84/enabled HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:60127 "POST /session/25f68880dc87aabc585fa24771588f40/element/f.A5818C05F378E5E09474B99C208F9AA5.d.B78F4AFEB072FF7A7008872CAE8C1035.e.84/click HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:60127 "GET /session/25f68880dc87aabc585fa24771588f40/url HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:60127 "GET /session/25f68880dc87aabc585fa24771588f40/title HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:60127 "DELETE /session/25f68880dc87aabc585fa24771588f40 HTTP/1.1" 200 0